ARSIS-Telemetry-GroundControl/groundcontrol-db/database.sql

```diff
@@ -31,4 +31,15 @@ CREATE TABLE IF NOT EXISTS biometrics (
   updatedAt TIMESTAMP DEFAULT now()
 );
 
+CREATE TABLE IF NOT EXISTS uia (
+  uuid uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+  id INTEGER NOT NULL,
```

Comment:
This should be the same as the other tables though it makes the insert fail, we should instead insert into this a default user too I think "user1" would be a good user1 string because its what the unity looks for anyway to register and we can have it start to register a next value if that is found so we can start building the event manager to be able to automatically figureout who it should register as.